.chronus/changes/lsp-2025-4-6-10-5-13.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: feature
+packages:
+  - "@typespec/compiler"
+  - typespec-vscode
+---
+
+leverage lsp to emit code instead of cli
```

Comment:
Split this changelog in 2 for the 2 packages as its not a comment relevant to the compiler.

```
Use language server to compile project instead of CLI 
```

```
[LSP] Expose new compile project command
```